content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -79,7 +79,7 @@ For a reference to old feature gates that are removed, please refer to
 | `CSIMigrationRBD` | `false` | Alpha | 1.23 | |
 | `CSINodeExpandSecret` | `false` | Alpha | 1.25 | |
 | `CSIVolumeHealth` | `false` | Alpha | 1.21 | |
-| `ComponentSLIs` | `false` | Alpha | 1.26 | |
+| `ComponentSLIs` | `true` | Beta | 1.27 | |
```

Comment:
```suggestion
| `ComponentSLIs` | `false` | Alpha | 1.26 | 1.26 |
| `ComponentSLIs` | `true` | Beta | 1.27 | |
```

Reply:
Was this gate a thing during 1.26?
We are not supposed to eliminate the trace that the gate was Alpha during 1.26 release.
You did not fix this and you did not reply with a different opinion, but you mark this conversation resolved?

Reply:
Oh I thought I accepted the suggestion.